core-bundle/src/Resources/contao/library/Contao/Automator.php

```diff
@@ -194,9 +188,13 @@ public function purgePageCache()
 
 	/**
 	 * Purge the search cache
+	 *
+	 * @deprecated Deprecated since Contao 4.0, to be removed in Contao 5.0.
```

Comment:
```suggestion
	 * @deprecated Deprecated since Contao 4.12, to be removed in Contao 5.0.
```

Reply:
Fixed in 4116d124af8d2e63077846cdad61a147e3c87045.